pkg/apis/core/v1alpha1/zz_generated.kubebuilder.go

```diff
@@ -1397,9 +1399,11 @@ var (
 				Plural: "propagatedversions",
 			},
 			Scope: "Namespaced",
+			Subresources: &v1beta1.CustomResourceSubresources{
```

Comment:
Might need a similar fix for MultiClusterServiceDNSRecord API object too. yeah the PR is not merged yet :)